allium/lib/search_discovery.py

```
@@ -0,0 +1,128 @@
+        ET.SubElement(entry, f"{{{SITEMAP_NAMESPACE}}}loc").text = url
+    return root
+
+
+def _remove_old_shards(output_path):
+    for candidate in output_path.glob("sitemap-*.xml"):
+        suffix = candidate.stem[len("sitemap-"):]
+        if suffix.isdigit() and candidate.is_file():
+            candidate.unlink()
+
+
+def _remove_search_discovery(output_path):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_remove_search_discovery`

Add return type annotation: `None`

(ANN202)